modules/page-memory/src/test/resources/META-INF/services/org.apache.ignite.internal.pagememory.io.PageIoModule

```diff
@@ -15,3 +15,4 @@
 # limitations under the License.
 #
 org.apache.ignite.internal.pagememory.TestPageIoModule
+org.apache.ignite.internal.pagememory.PageMemoryIoModule
```

Comment:
Wait a minute, why did you add this here?

Reply:
Yes, he is superfluous here, removed.